src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import cn from 'classnames';
+import { Link, useLocation } from 'react-router-dom';
+
+import {
+  usePeopleDispatch,
+  usePeopleState,
+} from '../components/store/PeopleContext';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+
+export const PeoplePage: React.FC = () => {
+  const location = useLocation();
```

Comment:
I think we can just use the destruction assignment here
```suggestion
  const { pathname } = useLocation();
```

Reply:
That is a mistake, because I am calling startsWith directly on the result of the useLocation() call, instead of on the pathname property of that object.